modules/node_modules/@ncigdc/components/Case.js

```diff
@@ -51,9 +51,9 @@ const enhance = compose(
 
 const Case = ({
   node,
-  totalNumCases,
   theme,
   numCasesAggByProject,
+  totalNumCases = Object.keys(numCasesAggByProject).reduce((sum, b) => sum + numCasesAggByProject[b], 0),
```

Comment:
Seeing as this isn't going to be passed in, might as well take it out of the props.